db/Schema.dbml

```diff
@@ -132,6 +132,7 @@ table waypoints {
   id int [ pk, not null, increment ]
   fk_route_id int
   fk_port_id int
+  sequence_id int     // counts from 0 and up every time a new set of waypoints is added to a route
```

Comment:
One last thing: Since you updated the DBML, you have to go through again and copy that to dbiagram.io, take a screenshot of the diagram and upload it and update the link in the readme to the latest diagram. I know it's a little tedious, but it's really helpful to always have the diagram up to date.